pkg/util/arangod/client.go

```diff
@@ -200,6 +218,22 @@ func createArangodHTTPConfigForDNSNames(ctx context.Context, cli corev1.CoreV1In
 	return connConfig, nil
 }
 
+// createArangodVSTConfigForDNSNames creates a go-driver VST connection config for a given DNS names.
+func createArangodVSTConfigForDNSNames(ctx context.Context, cli corev1.CoreV1Interface, apiObject *api.ArangoDeployment, dnsNames []string) (http.ConnectionConfig, error) {
+	scheme := "vst"
+	if apiObject != nil && apiObject.Spec.IsSecure() {
+		// TODO set security for transport
```

Comment:
Secure connection is essential, since almost all kube-arangodb deployments use TLS.
To use a secure VST connection, set `scheme = "https"`.